### DISCUSSION TITLE
Using _only_ custom Suricata rules?

### DISCUSSION THREAD

**Discussion by InfosecGoon** [Security-Onion-Solutions/securityonion#3679] [Unsupported Versions] [answered]
Is there an easy way to disable the ET OPEN rule download and use ONLY the custom rules that we're importing from our MISP instance via Wes's integration script? I was able to do this with the pulledpork configuration in Security Onion 16.04, but I can't figure out how to get idstools to comply in Security Onion 2.3.30. I thought that "so-rule disabled add re:sid:2" would disable anything with a SID starting with 2, but it doesn't seem to do anything.

Thanks for your time,

--Matt

**Comment by weslambert** [comment, marked as answer]
You can modify `/opt/so/saltstack/local/salt/idstools/etc/rulecat.conf` to remove the emerging-all.rules entry.

Alternatively, you could disable them, with something like:
```
disabled:
      - 're:ET '
```
